src/vs/workbench/contrib/chat/browser/actions/chatContextActions.ts

```diff
@@ -245,6 +267,17 @@ class AttachContextAction extends Action2 {
 					icon: pick.icon,
 					isTool: true
 				});
+			} else if ('kind' in pick && pick.kind === 'image') {
+				const fileBuffer = await clipboardService.readImage();
+				const uniqueId = fileBuffer.slice(0, 50).toString();
+				toAttach.push({
+					id: uniqueId,
+					name: localize('pastedImage', 'Pasted Image'),
+					fullName: 'Pasted Image',
```

Comment:
`fullName` too? I'm not sure the difference

Reply:
`fullName` doesn't get used anywhere else, but we do show `attachment.name` as a label

Reply:
It looks like it's intent is to be shown to the user:

https://github.com/microsoft/vscode/blob/3ae6c5ef651585b12870828230f214eed2e6ef07/src/vs/workbench/contrib/chat/browser/chatInputPart.ts#L641

I'm guessing this should either match `name` exactly (localized) or be undefined.

Reply:
ope yes forgot about that one 😭